README.md

```diff
@@ -13,12 +13,12 @@ Assuming you have a recent version of rust and cargo (v1.51.0+) installed
 (via [rustup](https://rustup.rs/)),
 then the following should get you a new repo to start a contract:
 
-First, install
-[cargo-generate](https://github.com/ashleygwilliams/cargo-generate).
+
+Install [cargo-generate](https://github.com/ashleygwilliams/cargo-generate) and cargo-run-script.
```

Comment:
Hmm, can't we use a native way to do thart? Like the aliases defined in `.cargo/config`

Reply:
Aliases only let you use cargo subcommands, not call shell scripts.

Adding `optimize.sh` or a Makefile are two alternative ways that depend on a more common tool (bash / make). But we do need a tool

Reply:
(I have tried to do more with cargo aliases before)

Reply:
Ah, right, thanks